src/oauthClients.json

```diff
@@ -0,0 +1,5 @@
+[{
```

Comment:
the clearest explanation what  client_secret and clientId are and what them from I found there:
https://docs.particle.io/guide/how-to-build-a-product/authentication/
in two words: they needed for the server to make sure that the client( particle cli or our web app or ...) which trying to interact with it is the particular, allowed client.